vue-press/src/.vuepress/sidebar.ts

```diff
@@ -28,62 +28,78 @@ export default sidebar({
             ]
         }
     ],
+    '/test/': [
+        {
+            text: 'Foo',
+            link: 'gg'
+        }
+    ],
     '/modules/': [
+        '',
+        {
+            text: 'HTTP',
+            link: 'http/',
+        },
+        {
+            text: 'Context',
+            link: 'context/',
+        },
+        'app/',
         {
-            text: 'Modules',
-            prefix: '/modules/',
+            text: 'Event',
+            prefix: 'event/',
+            link: 'event/',
             children: [
-                '',
                 {
-                    text: 'HTTP',
-                    link: 'http/',
+                    text: 'module',
+                    link: 'README.md',
                 },
                 {
-                    text: 'Context',
-                    link: 'context/',
+                    text: 'React',
+                    link: 'react.md',
                 },
-                'app/',
-                {
-                    text: 'Event',
-                    prefix: 'event/',
-                    link: 'event/',
-                    children: [
-                        {
-                            text: 'module',
-                            link: 'README.md',
-                        },
-                        {
-                            text: 'React',
-                            link: 'react.md',
-                        },
-                    ]
+            ]
 
+        },
+        {
+            text: 'Feature flag',
+            prefix: 'feature-flag/',
+            link: 'feature-flag/',
```

Comment:
The landing page for modules/feature-flag/ has no content, the landing page should be the child `module`

Reply:
default link for menu item is now module.md